openassessment/workflow/test/test_api.py

```diff
@@ -64,7 +64,7 @@ def test_create_workflow(self, data):
         self.assertEqual(
             workflow_keys,
             {
-                'submission_uuid', 'uuid', 'status', 'created', 'modified', 'score'
+                'submission_uuid', 'uuid', 'status', 'created', 'modified', 'score', 'override_score',
```

Comment:
Assuming we don't care about order (this is a literal `Set`, right?), we can minimize the diff by adding the new value on a separate line above the current one.

```python
{
    'override_score',
    'submission_uuid', 'uuid', 'status', 'created', 'modified', 'score'
}